modules/lightning_features/lightning_media/lightning_media.module

```diff
@@ -345,12 +397,8 @@ function lightning_media_ajax_render_alter(array &$data) {
       }
     }
 
-    // Gingerly check if the AJAX command class exists, so that we don't need
-    // to establish a hard dependency on CKEditor.
-    $command_class = '\Drupal\ckeditor\Ajax\AddStyleSheetCommand';
-    if ($style_sheets && class_exists($command_class)) {
-      /** @var \Drupal\ckeditor\Ajax\AddStyleSheetCommand $command */
-      $command = new $command_class($query->get('editor'), $style_sheets);
+    if ($style_sheets) {
+      $command = new AddStyleSheetCommand($query->get('editor'), $style_sheets);
```

Comment:
This is gardening. Lightning Media has a hard dependency on the CKEditor module, so there was no need to gingerly check anything.